src/postgres/src/test/regress/expected/yb_pg_partition_aggregate.out

```diff
@@ -0,0 +1,474 @@
+--
+-- PARTITION_AGGREGATE
+-- Test partitionwise aggregation on partitioned tables
+--
+-- Enable partitionwise aggregate, which by default is disabled.
+SET enable_partitionwise_aggregate TO true;
+-- Enable partitionwise join, which by default is disabled.
+SET enable_partitionwise_join TO true;
+-- Disable parallel plans.
+SET max_parallel_workers_per_gather TO 0;
+--
+-- Tests for list partitioned tables.
+--
+CREATE TABLE pagg_tab (a int, b int, c text, d int) PARTITION BY LIST(c);
+CREATE TABLE pagg_tab_p1 PARTITION OF pagg_tab FOR VALUES IN ('0000', '0001', '0002', '0003');
+CREATE TABLE pagg_tab_p2 PARTITION OF pagg_tab FOR VALUES IN ('0004', '0005', '0006', '0007');
+CREATE TABLE pagg_tab_p3 PARTITION OF pagg_tab FOR VALUES IN ('0008', '0009', '0010', '0011');
+INSERT INTO pagg_tab SELECT i % 20, i % 30, to_char(i % 12, 'FM0000'), i % 30 FROM generate_series(0, 2999) i;
+-- When GROUP BY clause matches; full aggregation is performed for each partition.
```

Comment:
I notice the EXPLAIN tests are removed from the original. Did they not work or were they unreliable?
I think it's good to have them here -- if they don't yet work we can comment them out and add a comment explaining what the limitation is

Reply:
Thanks for reminding me about these! Sorry that I missed them out. A good number of them don't work. I now added all of them back, and have left the ones that don't work commented out. If you could create another ticket and assign it to me, I will take care of those as part of a separate task after this one is pushed.
While exploring this I noticed I missed out partition_prune tests. Added those as well.